shared/src/common/enums/user/user-validation-message.enum.ts

```diff
@@ -1,17 +1,17 @@
 import { UserValidationRule } from '../validation/validation.enum';
 
 const UserValidationMessage = {
-  NAME_REQUIRE: 'Username is required',
-  NAME_MIN_LENGTH: `Username must be at least ${UserValidationRule.NAME_MIN_LENGTH} characters long`,
-  NAME_MAX_LENGTH: `Username must be at most ${UserValidationRule.NAME_MAX_LENGTH} characters long`,
+  NAME_REQUIRE: 'Full name is required',
+  NAME_MIN_LENGTH: `Full name must be at least ${UserValidationRule.NAME_MIN_LENGTH} characters long`,
```

Comment:
You can use `{#limit}` instead of `${UserValidationRule.NAME_MIN_LENGTH}`. Joi will automatically pass the value. Same for other cases also

Reply:
Good catch, but let's use an enum for it

Reply:
Do you mean we should use `UserValidationRule`? In that case, I was getting `UserValidationRule` as undefined, not sure why